### ISSUE TITLE
Runfiles are not embedded when `--build_runfile_manifests=false`

### ISSUE THREAD

**Issue by cerisier** [bazel-contrib/bazel-lib#936]
I believe this is due to the way the tar rule computes the runfiles_dir.

https://github.com/bazel-contrib/bazel-lib/blob/main/lib/private/tar.bzl#L227-L230

Isn't `default_info.files_to_run.executable + ".runfiles"` sufficient ?

Happy to submit a PR.

**Comment by cerisier** [comment]
Maybe @fmeum would know something about that.

**Comment by fmeum** [comment]
I agree that creating a `.runfiles` directory for each executable is the correct approach. Runfiles are never empty as they always at least contain the executable itself. The logic that inspects `runfiles_manifest` can be removed.

Happy to review a PR!
